### DISCUSSION TITLE
Welcome to torchtune discussions!

### DISCUSSION THREAD

**Discussion by joecummings** [pytorch/torchtune#957] [Announcements]
Hey there! This is a space for our community to connect and engage with each other. Here, you can:

- Ask any questions related to torchtune
- Share your ideas and receive feedback from the community

For bugs/feature request, please post them in Issues tab instead with an appropriate amount of information. 

We look forward to your contributions. Happy tuning!

**Comment by mariozupan** [comment]
Very nice work, very intuitive. Works well on two GPUs full fine tuning. However I would like to know how to make train validation split and then to have train/validation loss in my logs?

**Reply by ebsmothers** [reply]
Hi @mariozupan just saw your comment -- this should be possible now that #2464 has landed. You can also see #2608 as an example of the config changes you should make to do this.
